lib/db.js

```diff
@@ -827,6 +827,24 @@ module.exports = function (
     return this.pool.del('/verificationReminders', reminderData)
   }
 
+  DB.prototype.securityEvent = function (event) {
```

Comment:
Was just following style of this module to name *creator* methods as the object they are creating :D